lib/src/app/app.dart

```diff
@@ -16,25 +17,151 @@ class AppClient {
+    return response.organizationIdentities;
+  }
+
+  /// List the [Organization]s a user belongs to
+  Future<List<OrgDetails>> listOrganizationsByUser(String userId) async {
+    final request = ListOrganizationsByUserRequest()..userId = userId;
+    final ListOrganizationsByUserResponse response = await _client.listOrganizationsByUser(request);
+    return response.orgs;
+  }
+
   /// Get a specific [Organization] by ID
   Future<Organization> getOrganization(String organizationId) async {
     final getOrganizationRequest = GetOrganizationRequest()..organizationId = organizationId;
     final GetOrganizationResponse response = await _client.getOrganization(getOrganizationRequest);
     return response.organization;
   }
 
-  /// List the [Location] of a specific [Organization] that the currently authenticated user has access to
-  Future<List<Location>> listLocations(Organization organization) async {
-    final listLocationsRequest = ListLocationsRequest()..organizationId = organization.id;
-    final ListLocationsResponse response = await _client.listLocations(listLocationsRequest);
-    return response.locations;
+  /// Checks for namespace availablity throughout all [Organization]s.
+  Future<bool> getOrganizationNamespaceAvailability(String publicNamespace) async {
+    final request = GetOrganizationNamespaceAvailabilityRequest()..publicNamespace = publicNamespace;
+    final GetOrganizationNamespaceAvailabilityResponse response = await _client.getOrganizationNamespaceAvailability(request);
+    return response.available;
+  }
+
+  /// Update an [Organization]
+  Future<Organization> updateOrganization(String organizationId,
+      {String? name, String? publicNamespace, String? region, String? cid}) async {
+    final request = UpdateOrganizationRequest()
+      ..organizationId = organizationId
+      ..name = name ?? ''
+      ..publicNamespace = publicNamespace ?? ''
+      ..region = region ?? ''
+      ..cid = cid ?? '';
+    final UpdateOrganizationResponse response = await _client.updateOrganization(request);
+    return response.organization;
+  }
+
+  /// Delete an [Organization]
+  Future<void> deleteOrganization(String organizationId) async {
+    final request = DeleteOrganizationRequest()..organizationId = organizationId;
+    await _client.deleteOrganization(request);
+  }
+
+  /// List the members and pending invites for an [Organization].
+  Future<ListOrganizationMembersResponse> listOrganizationMembers(Organization org) async {
+    final request = ListOrganizationMembersRequest()..organizationId = org.id;
+    final ListOrganizationMembersResponse response = await _client.listOrganizationMembers(request);
+    return response;
+  }
+
+  /// Send an invitation to to join an [Organization] to the specified email. Grant the level of permission defined in the [ViamAuthorization] object attached.
+  Future<OrganizationInvite> createOrganizationInvite(Organization org, String email, List<ViamAuthorization> authorizations,
```

Comment:
See above re: `org` param versus `orgId` param.

edit: it looks like this happens in a number of cases below as well. Can we replace all insances of an `org` or `location` parameter where we just grab an orgId or locationId with an `orgId` or `locationId` parameter?

Reply:
I definitely started this back in the day. I agree that passing IDs is the easiest way to do it and should probably be switched

Reply:
Yep, will do. These were the original functions that were already in the client.